sybilCredentialAtomicSig.go

```diff
@@ -0,0 +1,289 @@
+	IssuerAuthClaimMtp   []*merkletree.Hash `json:"issuerAuthClaimMtp"`
+	IssuerAuthClaimsRoot *merkletree.Hash   `json:"issuerAuthClaimsRoot"`
+	IssuerAuthRevRoot    *merkletree.Hash   `json:"issuerAuthRevRoot"`
+	IssuerAuthRootsRoot  *merkletree.Hash   `json:"issuerAuthRootsRoot"`
+
+	IssuerAuthClaimNonRevMtp      []*merkletree.Hash `json:"issuerAuthClaimNonRevMtp"`
+	IssuerAuthClaimNonRevMtpAuxHi *merkletree.Hash   `json:"issuerAuthClaimNonRevMtpAuxHi"`
+	IssuerAuthClaimNonRevMtpAuxHv *merkletree.Hash   `json:"issuerAuthClaimNonRevMtpAuxHv"`
+	IssuerAuthClaimNonRevMtpNoAux string             `json:"issuerAuthClaimNonRevMtpNoAux"`
+
+	IssuerClaim                 *core.Claim      `json:"issuerClaim"`
+	IssuerClaimNonRevClaimsRoot *merkletree.Hash `json:"issuerClaimNonRevClaimsRoot"`
+	IssuerClaimNonRevRevRoot    *merkletree.Hash `json:"issuerClaimNonRevRevRoot"`
+	IssuerClaimNonRevRootsRoot  *merkletree.Hash `json:"issuerClaimNonRevRootsRoot"`
+
+	IssuerClaimNonRevState    *merkletree.Hash   `json:"issuerClaimNonRevState"`
+	IssuerClaimNonRevMtp      []*merkletree.Hash `json:"issuerClaimNonRevMtp"`
+	IssuerClaimNonRevMtpAuxHi *merkletree.Hash   `json:"issuerClaimNonRevMtpAuxHi"`
+	IssuerClaimNonRevMtpAuxHv *merkletree.Hash   `json:"issuerClaimNonRevMtpAuxHv"`
+	IssuerClaimNonRevMtpNoAux string             `json:"issuerClaimNonRevMtpNoAux"`
+
+	IssuerClaimSignatureR8X string `json:"issuerClaimSignatureR8x"`
+	IssuerClaimSignatureR8Y string `json:"issuerClaimSignatureR8y"`
+	IssuerClaimSignatureS   string `json:"issuerClaimSignatureS"`
+
+	ClaimSchema string `json:"claimSchema"`
+
+	StateCommitmentClaim           *core.Claim        `json:"stateCommitmentClaim"`
+	StateCommitmentClaimMtp        []*merkletree.Hash `json:"stateCommitmentClaimMtp"`
+	StateCommitmentClaimClaimsRoot *merkletree.Hash   `json:"stateCommitmentClaimClaimsRoot"`
+	StateCommitmentClaimRevRoot    *merkletree.Hash   `json:"stateCommitmentClaimRevRoot"`
+	StateCommitmentClaimRootsRoot  *merkletree.Hash   `json:"stateCommitmentClaimRootsRoot"`
+	StateCommitmentClaimIdenState  *merkletree.Hash   `json:"stateCommitmentClaimIdenState"`
+
+	GistRoot     *merkletree.Hash   `json:"gistRoot"`
+	GistMtp      []*merkletree.Hash `json:"gistMtp"`
+	GistMtpAuxHi *merkletree.Hash   `json:"gistMtpAuxHi"`
+	GistMtpAuxHv *merkletree.Hash   `json:"gistMtpAuxHv"`
+	GistMtpNoAux string             `json:"gistMtpNoAux"`
+
+	CRS *big.Int `json:"crs"`
+
+	// user data
+	UserGenesisID            string `json:"userGenesisID"`
+	ProfileNonce             string `json:"profileNonce"`
+	ClaimSubjectProfileNonce string `json:"claimSubjectProfileNonce"`
+
+	RequestID *big.Int `json:"requestID"`
+	IssuerID  string   `json:"issuerID"`
+	Timestamp int64    `json:"timestamp"`
+}
+
+func (s SybilAtomicSigInputs) Validate() error {
+	if s.ID == nil {
+		return errors.New(ErrorEmptyID)
+	}
+
+	if s.GISTProof.Proof == nil {
+		return errors.New(ErrorEmptyGlobalProof)
```

Comment:
Again, replace the word **Global** names with **GIST** maybe? Do it here, and search and replace it in all other places. It's harder to maintain when we have a naming  ambiguity.

Reply:
Ok, I've updated everywhere in this repo (Authv2 and other unrelated circuits as well), from Global to GIST